app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb

```diff
@@ -34,17 +34,20 @@ def medications?
     end
 
     def add_intro
-      names = @patient.with_indifferent_access
-      full_name = [names[:first], names[:middle], names[:last]].reject(&:blank?).join ' '
-      patient_name = [full_name, names[:suffix]].reject(&:blank?).join ', '
+      patient_info = @patient.with_indifferent_access
```

Comment:
Sorry but please update to `@patient_info`. You'll have to update `initialize` method as well as the parameter name to [`pdf`](https://github.com/department-of-veterans-affairs/vets-api/pull/9354/files#diff-81632810da9fc418d0574944b4d877b7f48ca95fa375d87d23691941b5a4af38R102). It's the right thing to do since the parameter is no longer simply a `full_name`.
```suggestion
      patient_info = @patient_info.with_indifferent_access
```

Reply:
Forgot this one.